benchmarking/daint_single_node/stdout_to_json.py

```diff
@@ -115,14 +115,54 @@ def stdout_to_json(stdout_file_regex, run_directory):
     match = re.search(r"^GIT_COMMIT = ([a-z0-9]+)$", git_env, re.MULTILINE)
     if match:
         setup["git hash"] = match.group(1)
+    return setup
+
 
-    # print as JSON
+def print_to_output(setup: Dict[str, Any], times: Dict[str, Any], output=sys.stdout):
+    """Prints the collected data into the specified output"""
     experiment = {}
     experiment["setup"] = setup
     experiment["times"] = times
-    json.dump(experiment, sys.stdout, indent=4)
+    json.dump(experiment, output, indent=4)
     print("")
```

Comment:
```suggestion
```

Reply:
This just make sure the HTML has a newline in the end of the report